### PR TITLE
Remove disk ledger dir on bootstrap.

### PR DESCRIPTION
**Pull request by Eitu33**

### CODE REVIEW

File: massa-node/src/main.rs [hunk 1]
```diff
@@ -99,6 +99,13 @@ async fn launch() -> (
         FinalState::new(final_state_config).expect("could not init final state"),
     ));
 
+    // Remove current disk ledger if there is one
+    // NOTE: this is temporary, since we cannot currently handle bootstrap from remaining ledger
+    if SETTINGS.ledger.disk_ledger_path.exists() {
```
Comment (AurelienFT) [inline massa-node/src/main.rs:104]: Isn't a problem that we delete after getting a connector in the initialization of final state just above ? The new DB will still exist or we should delete it before the final state ?

**Review by AurelienFT — APPROVED** [review]

File: massa-node/src/main.rs (hunk 1 above)
Comment (Eitu33) [reply massa-node/src/main.rs:100]: Well spotted, could've been a nasty mistake.

**Comment by AurelienFT** [comment]
bors merge

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**